### DISCUSSION TITLE
mac0S 10.13 Problem with - brew install

### DISCUSSION THREAD

**Discussion by bobbyfisher777** [Homebrew/discussions#1356] [Getting started]
Hi everyone, I'm a beginner I need help. I successfully downloaded homebrew after I downloaded Xcode 9.4.1 version for my macOS 10.13. I try to run the command "brew install + package and it doesn't work." Says
---Warning: You are using macOS 10.13.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version. -------------attempts to download but eventually adds ---Warning: Your Xcode (9.4.1) is outdated.
Please update to Xcode 10.1 (or delete it).
Xcode can be updated from the App Store.
I would be very grateful if anyone could help me ,, thanks a lot guys.

**Comment by SMillerDev** [comment]
>  We (and Apple) do not provide support for this old version.

This is your issue, if you can't update your macOS version we can't really help you.

> Please create pull requests instead of asking for help on Homebrew's GitHub, Twitter or any other official channels. You are responsible for resolving any issues you experience while you are running this old version. 

This also means that this question isn't really suited here.

**Reply by bobbyfisher777** [reply]
ok i thought you could use homebrew with my mac version too. Thanks for the information.

**Reply by Bo98** [reply]
It's possible but it's just not actively supported and you'll have to build many things from source rather than rely on pre-builts.

These are warnings rather than hard errors, that are there just to let you know that you're on your own if something breaks.

**Comment by lantren** [comment]
This is the answer. 
> Please update to Xcode 10.1 (or delete it).

I realize this is old but in case someone else gets unhelpful suggestions about this problem. 

try to delete your old version of Xcode and don't have Xcode installed at all.   Brew will install,  at least it did for me.  I am running 10.14.6 and got the same Xcode too old error.

**Reply by gromgit** [reply]
In the OP's case, they _must_ have Xcode installed, because almost all installs would be build-from-source.

A more useful approach is to look up the latest Xcode release for your macOS version on https://xcodereleases.com/, then download and install the Command Line Tools. That's a lot lighter than the full Xcode package, sufficient to build most formulae, and is required at runtime by certain key formulae like Python, if I remember correctly.

**Reply by lantren** [reply]
I must have had command line tools installed already.  The version check must be looking at  the version of the Xcode .app itself.  All i know is , i deleted it and I was no longer stopped by the "Warning: Your Xcode (X.X.X) is outdated" error.

**Reply by gromgit** [reply]
Yes, Xcode and Command Line Tools are two separate packages that are warned about separately.

Check your `brew config` output. `CLT` refers to Command Line Tools, while `Xcode` means the full package.
